tests/suites/0_stateless/05_ddl/05_0006_ddl_grant_privilege_v2.sql

```diff
@@ -0,0 +1,36 @@
+DROP USER IF EXISTS 'test-grant'@'localhost';
```

Comment:
```suggestion
set enable_planner_v2 = 1;
DROP USER IF EXISTS 'test-grant'@'localhost';
```

Reply:
Fixed.

Reply:
'root'@mysqldb 00:52:33 [(none)]> SET enable_planner_v2 = 1;
Query OK, 0 rows affected (0.05 sec)
Read 0 rows, 0.00 B in 0.002 sec., 0 rows/sec., 0.00 B/sec.

'root'@mysqldb 00:52:36 [(none)]> CREATE TABLE IF NOT EXISTS `db01`.`tb1`;
ERROR 1105 (HY000): Code: 1002, displayText = Unsupported CREATE TABLE statement.


Because of this err , I rollback my modify.

Reply:
> ```sql
> CREATE TABLE IF NOT EXISTS `db01`.`tb1`;
> ```

This test looks wiered, since it doesn't specify table definition. Maybe it is expected to be failed?

Reply:
> ```sql
> ```sql
> CREATE TABLE IF NOT EXISTS `db01`.`tb1`;
> ```
> 
> 
>     
>       
>     
> 
>       
>     
> 
>     
>   
> 
> This test looks strange, since it doesn't specify table definition. Maybe it is expected to be failed?
> ```

In my mind I think that’s a bad case. No sense.

Reply:
Modified.